### PR TITLE
Sync on lbry.tv

### PR DESCRIPTION
**Pull request by seanyesmunt**
Still requires more testing I would say. It is currently deployed to `dev.lbry.tv`.

### CODE REVIEW

**Comment by seanyesmunt** [comment]
@tzarebczan Can you give this a try?

File: src/ui/util/saved-passwords.js [hunk 1]
```diff
@@ -37,16 +37,23 @@ function deleteCookie(name: string) {
...
     const password = value === undefined || value === null ? '' : value;
     sessionPassword = password;
 
     if (saveToDisk) {
       if (password) {
+        // @if TARGET='app'
         ipcRenderer.send('set-password', password);
+        // @endif
+        // @if TARGET='web'
+        setCookie('saved-password', password, 14);
```
Comment (kauffj) [inline src/ui/util/saved-passwords.js]: I should have caught this in an earlier review, but it was only reading this that triggered me. A lot of this file speaks generically of "saved password" or "saved passwords", but in reality this file only allows for the setting of a single password and a very specific one at that: your wallet password.

File: src/ui/util/lbrytv.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
+const { LBRY_TV_API } = require('../../../config');
+
 function generateStreamUrl(claimName, claimId) {
-  return `https://api.lbry.tv/content/claims/${claimName}/${claimId}/stream`;
+  const prefix = process.env.SDK_API_URL || LBRY_TV_API;
```
Comment (kauffj) [inline src/ui/util/lbrytv.js]: What's the situation where setting `process.env.SDK_API_URL` is insufficient?

File: src/ui/util/saved-passwords.js (hunk 1 above)
Comment (seanyesmunt) [reply src/ui/util/saved-passwords.js]: Also auth_token

**Pull request merged**